docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+- [Microsoft.NETCore.Portable.Compatibility](https://www.nuget.org/packages/Microsoft.NETCore.Portable.Compatibility) - A set of compatibility facades that enable mscorlib-based Portable Class Libraries (PCLs) to run on .NET Core.
+
+The .NET Standard Library metapackage:
+
+- [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) - Describes the libraries that are part of the ".NET Standard Library". Applies to all .NET implementations (for example, .NET Framework, .NET Core and Mono) that support the .NET Standard Library. Establishes the 'netstandard' framework.
+
+Metapackages are referenced just like any other NuGet package in project.json. 
+
+In the following example, the `NETStandard.Library` meta package is referenced, which is used for creating libraries that are portable across .NET runtimes.
+
+```json
+{
+  "dependencies": {
+    "NETStandard.Library": "1.5.0"
+  },
+  "frameworks": {
+    "netstandard1.5": {}
+  }
+}
+```
+
+In the following example, the `Microsoft.NETCore.App` metapackage is referenced, which is used for creating apps and libraries that are intended to run on and take full advantage of .NET Core. It provides access to a larger set of libraries than are provided by `NETStandard.Library`.
+
+```json
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0"
+  },
+  "frameworks": {
+    "netcoreapp": {}
+  }
+}
+```
+
+Target Frameworks
+=================
+
+.NET Core packages each support a set of target frameworks, declared with target framework folders (within the lib and ref folders mentioned earlier). Target frameworks describe an available API set (and potentially other characteristics) that you can rely on when you target a given framework. They are versioned as new APIs are added.
+
+For example, [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) supports the following target frameworks:
+
+- net46
+- netstandard1.3
+- 6 different Xamarin platforms (for example, xamarinios10)
+
+It is useful to contrast the first two of these target frameworks, since they are examples of the two different ways that target frameworks are defined.
+
+The `net46` framework represents the available APIs in the .NET Framework 4.6. You can produce libraries  compiled with the .NET Framework 4.6 reference assemblies and then distribute those libraries in NuGet packages in a net46 lib folder. It will be used for apps that target the .NET Framework 4.6 or that are compatible with it. This is how all framework targets have traditionally worked.
+
+The `netstandard1.3` framework is a package-based target framework, so it doesn't expose or describe any APIs in the absence of packages that target that framework. It is the interplay of the framework and the packages that define the set of APIs that are available when targeting the framework.
```

Comment:
> It is the interplay of the framework and the packages that define the set of APIs that are available when targeting the framework.

How is this interplay defined?  I'm interpreting this as a statement that I won't really be able to reason about why things act the way they do.